docs/docs/about/comparison-powershell.md

```diff
@@ -307,7 +307,7 @@ Enable-PnPCommSite|PnP.PowerShell|[spo site commsite enable](../cmd/spo/site/sit
 Enable-PnPFeature|PnP.PowerShell|[spo feature enable](../cmd/spo/feature/feature-enable.md)
 Enable-PnPFlow|PnP.PowerShell|[spo flow enable](../cmd/flow/flow-enable.md)
 Enable-PnPPageScheduling|PnP.PowerShell|
-Enable-PnPPowerShellTelemetry|PnP.PowerShell|
+Enable-PnPPowerShellTelemetry|PnP.PowerShell|[cli config set](../telemetry/#disable-telemetry)
```

Comment:
Hi @Adam-it, just did a quick check. Seems like you are targeting the wrong section here. It should reference to `#re-enable-telemetry`.

Reply:
sorry. didn't notice that section 🤦